iosApp/iosApp/Utils/RouteTypeExtension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RouteTypeExtension.swift
+//  iosApp
+//
+//  Created by Jack Curtis on 10/8/24.
+//  Copyright © 2024 MBTA. All rights reserved.
+//
+
+import Foundation
+import shared
+
+extension RouteType {
```

Comment:
question: Could vehicleTypeText in UpcomingTripView be updated to use this as well?

Reply:
a7443ead